artiq/gateware/eem.py

```diff
@@ -38,20 +38,32 @@ def io(eem, iostandard="LVDS_25"):
             for i in range(8)]
 
     @classmethod
-    def add_std(cls, target, eem, ttl03_cls, ttl47_cls, iostandard="LVDS_25"):
+    def add_std(cls, target, eem, ttl03_cls, ttl47_cls, iostandard="LVDS_25",
+            add_edge_counters=False):
```

Comment:
Keeping this in line with the other arguments and the other eems, I'd pass the class to use and not a Boolean. That paves the way for a future counter phy that handles serdes data.